packages/commerce-server/src/format-prices-for-product.ts

```diff
@@ -103,7 +133,7 @@ export async function formatPricesForProduct(
           status: true,
         },
       })
-    : false
+    : null
```

Comment:
This makes the types easier -- `Purchase | null` instead of `Purchase | null | false`